dev/docker-clients/client.nomad

```diff
@@ -0,0 +1,52 @@
+job "client" {
+  datacenters = ["dc1"]
+
+  group "client" {
+    count = 6
+
+    task "agent" {
+      driver = "docker"
+
+      config {
+        image = "djenriquez/nomad:v0.6.0"
+
+        # command = "nomad"
+        args         = ["agent"]
+        network_mode = "host"
+        volumes      = ["local/config:/etc/nomad", "/var/run/docker.sock:/var/run/docker.sock", "/tmp:/tmp"]
+        privileged   = true
+      }
+
+      resources {
+        cpu    = 300 # 500 MHz
+        memory = 100 # 256MB
+
+        network {
+          mbits = 10
+          port  "http"{}
+        }
+      }
+
+      template {
+        data = <<EOF
+log_level = "DEBUG"
+data_dir = "/tmp/client{{ env "NOMAD_ALLOC_INDEX" }}"
+name = "client-{{ env "NOMAD_ALLOC_INDEX" }}"
+client {
+	enabled = true
+	servers = ["127.0.0.1:4647"]
+	options {
+	  "driver.raw_exec.enable" = "1"
+    }
```

Comment:
whitespace is off. I've been trying to switch to 2 space indents based on some docs feedback from seth